cmd/acraserver/listener.go

```diff
@@ -84,15 +77,15 @@ func (server *SServer) handleConnection(connection net.Conn) {
 	}
 	clientSession, err := NewClientSession(server.keystorage, server.config, connection)
 	if err != nil {
-		log.WithError(err).Println("Error: can't initialize client session")
+		log.WithError(err).Println("can't initialize client session")
 		if closeErr := connection.Close(); closeErr != nil {
-			log.WithError(closeErr).Println("Error: can't close connection")
+			log.WithError(closeErr).Println("can't close connection")
 		}
 		return
 	}
 	clientSession.connection = wrappedConnection
 
-	log.Println("Debug: secure session initialized")
+	log.Debugln("secure session initialized")
```

Comment:
maybe `log.Infof`?

Reply:
on my opinion it's unnecessary info and useful only for debugging